otterdog/eclipse-opendut.jsonnet

```diff
@@ -33,6 +44,9 @@ orgs.newOrg('eclipse-opendut') {
           requires_conversation_resolution: true,
           requires_status_checks: false,
         },
+        orgs.newBranchProtectionRule('development') {
+          allows_deletions: false,
```

Comment:
I assume you want a branch protection rule that only prevents deletion of the branch?
This rule will not do exactly what you want as it will use the default values, you can go to the playground to adjust the rule to your needs: https://eclipse-opendut.github.io/.eclipsefdn/playground/

Here is an example:

![image](https://github.com/eclipse-opendut/.eclipsefdn/assets/1048055/3dbbb0ba-b2a4-4f23-a30b-7105c7dccf15)

you see that various setting are there even if you dont specify them, but you can change them ofc by overriding them in your snippet.

Reply:
Are you sure that the `allows_deletions` setting works as expected? Because i can delete any branch:
![Screenshot_20231214_144518](https://github.com/eclipse-opendut/.eclipsefdn/assets/11158578/581c80a5-b680-4908-9464-3d31b46533f3)

Reply:
the changes will only become active when this PR got merged and applied, till now there is no branch protection rule in place to prevent deletion of the development branch.

Reply:
so, yes you will need to add a branch protection rule for the development branch that prevents deletion. Your rule was ok I was just pointing out that apart from the setting the rule will also have some default values e.g. number of approvals of 2 which you might not want for the development branch so you have to adjust these settings with the playground.

Reply:
Ah. We misunderstood the meaning of 'default' in this case. :see_no_evil: Sorry. We thought that there is a branch protection rule (a default one) which prevents the deletion of a branch.

Thank you for your patience!

Reply:
no worries, I am here to help and I understand it might not be obvious from the start. Once you get the knack its actually a pretty handy tool imho.